_stbt/core.py

```diff
@@ -2196,6 +2196,13 @@ def _log_image_descriptions(
         ))
 
 
+def _iter_frames(depth=1):
+    frame = inspect.currentframe(depth + 1)
```

Comment:
`inspect.currentframe` is CPython only. I suppose that's OK as we're pretty much wedded to CPython 2.7 at this point.